src/fscacher/cache.py

```diff
@@ -200,14 +200,14 @@ def __init__(self):
         self.hash_ords = None
 
     def add_file(self, path, fprint: FileFingerprint):
```

Comment:
for paranoid me, could you please craft a test which would exercise the assumption that order of files would not matter for the fingerprint?

Reply:
Test added: a5f1e86